ticketutil/bugzilla.py

```diff
@@ -149,10 +207,12 @@ def _create_ticket_request(self, params):
         """
         # Attempt to create ticket.
         try:
-            if self.token:
-                params['token'] = self.token
+            #if self.token:
+            #    params['token'] = self.token
+            params.update(self.auth.getCredentials())
+            
 
-            r = self.s.post(self.rest_url, json=params)
+            r = self.s.post(self.rest_url, json=params, verify=False)
```

Comment:
Along the same lines as the previous comment about updating s.params, you can also set "s.verify = False" in _create_requests_session(), similar to how it's done in line 68 of ticket.py. This makes it so you don't need "verify=False" in every requests call throughout all the other methods.